src/sentry/templates/sentry/layout.html

```diff
@@ -71,6 +71,33 @@
   {% endblock %}
 </head>
 
+
+{% if show_session_replay_banner %}
+  <style type="text/css">
+    .alert-banner {
+      background: rgb(255, 219, 74);
+      padding-top: 0.75rem;
+      padding-bottom: 0.75rem;
+      display: flex;
+      justify-content: center;
+      border-radius: 4px;
+      margin-bottom: 4px;
+    }
+    .alert-message {
+      margin-left: 1rem;
+      margin-right: 1rem;
+      line-height: 1.5;
+      display: inline-flex;
+      align-items: center;
+      justify-content: center;
+      text-align: left;
+      flex-grow: 1;
+      font-size: 16px;
+      font-family: Rubik;
+    }
+  </style>
```

Comment:
Do we not have any existing CSS for this?

Reply:
@billyvg Not in HTML that's not used in emails. Not too many styles to choose from there.